src/components/Preferences/Availability/CourseSpread.tsx

```diff
@@ -0,0 +1,70 @@
+import { FormControl, FormLabel } from "@chakra-ui/react";
+import {
+    Select as MultiSelect,
+    ActionMeta,
+    MultiValue,
+    OptionBase,
+    GroupBase,
+} from "chakra-react-select";
+
+const labels = {
+    M: "Monday",
+    T: "Tuesday",
+    W: "Wednesday",
+    Th: "Thursday",
+    F: "Friday",
+    TWF: "TWF",
+    MTh: "MTh",
+};
+
+interface PreferredDayOption extends OptionBase {
+    label: string;
+    value: string;
+}
+
+const preferredDayOptions: PreferredDayOption[] = [
```

Comment:
I think TWF and MTH should be first because they are more likely to be chosen but not a big deal

Reply:
good idea